library/Fission/Internal/Orphanage/HigherOrder.hs

```diff
@@ -0,0 +1,18 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Fission.Internal.Orphanage.HigherOrder () where
+
+
+import Fission.Prelude
+
+import Servant
+import Servant.Swagger
+import Data.Swagger
+
+import Fission.Web.Auth.Types
+
+-- @@TODO: add an accurate security definition
```

Comment:
???

Reply:
Ah right. This is for Swagger. The problem is, none of the [Security Scheme](https://hackage.haskell.org/package/swagger2-2.5/docs/Data-Swagger.html#t:SecuritySchemeType) options match what we're doing. I guess the closest is ApiKey? And then we can describe it more in depth in the description